samples/mpu_stack_guard_test/src/main.c

```diff
@@ -65,6 +66,5 @@ void main(void)
 	printk("Canary Initial Value = 0x%x\n", buf.canary);
 
 	/* spawn stack_guard_thread */
-	k_thread_spawn(buf.stack, STACKSIZE, stack_guard_thread, NULL, NULL,
-		       NULL, PRIORITY, 0, K_NO_WAIT);
+	k_thread_create(&buf.thread, buf.stack, STACKSIZE, stack_guard_thread,			NULL,NULL, NULL, PRIORITY, 0, K_NO_WAIT);
```

Comment:
This was recently fixed in #478, but the arm branch hasn't been rebased yet to pick it up.